app/controllers/course/assessment/question/scribing_controller.rb

```diff
@@ -0,0 +1,155 @@
+# frozen_string_literal: true
+# rubocop:disable Metrics/ClassLength
+class Course::Assessment::Question::ScribingController < \
+  Course::Assessment::QuestionsController
+  load_and_authorize_resource :scribing_question,
+                              class: Course::Assessment::Question::Scribing,
+                              through: :assessment, parent: false
+
+  include CarrierWave::MiniMagick
+
+  def new
+    respond_to do |format|
+      format.html { render 'new' }
+    end
+  end
+
+  def show
+    respond_to do |format|
+      format.json { render_scribing_question_json }
+    end
+  end
+
+  def create # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+    # special handling for PDF files
+    if params&.[](:question_scribing)&.[](:file)&.content_type&.downcase == 'application/pdf'
```

Comment:
I just realised Ruby has a `dig` function - http://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig

Alternatively, you can place this in a method, and call this `if file_for_uploading?`, and extend when we support other file types.